cloudhub-2/modules/ROOT/pages/ps-config-app-level-egress.adoc

```diff
@@ -0,0 +1,167 @@
+= Configuring Application-Level Egress Rules
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Application-level egress rules are independent of private-space-level firewall rules. For egress traffic to go through, it must pass both private space and application-level control.
+
+
+== Before You Begin
+
+* Set up your private space and finish provisioning your private network.
+* Verify that your account has CloudHub Network Viewer permission, which is required to view a private space's firewall and application level rules.
+* Verify that your account has CloudHub Network Administrator permission, which is required to modify a private space's firewall and application level rules.
+
+== Configure Application Level Rules in Your Private Space
```

Comment:
Please change all occurrences of application level rules to application-level rules in text and headings.  Here is an article about compound adjectives: https://www.gingersoftware.com/content/grammar-rules/adjectives/compound-adjectives